ingestion/src/metadata/readers/dataframe/reader_factory.py

```diff
@@ -42,6 +42,9 @@ class SupportedTypes(Enum):
     JSON = "json"
     JSONGZ = "json.gz"
     JSONZIP = "json.zip"
+    JSONL = "jsonl"
+    JSONLGZ = "jsonl.gz"
+    JSONLZIP = "jsonl.zip"
```

Comment:
A thought: it might be more convenient to have a mapping from synonymous file extensions to their canonical types rather than enumerate all file extensions as separate types. There are a variety of other file extensions conventions that are synonymous (e.g. `.pq` and `.parquet`)